xrpl/wallet/main.py

```diff
@@ -16,23 +17,53 @@ class Wallet:
     details.
     """
 
+    @property
+    def address(self: Wallet) -> str:
+        """
+        Alias for wallet.classic_address.
+
+        Returns:
+            The address that publicly identifies this wallet, as a base58 string.
+        """
+        return self.classic_address
+
+    @address.setter
+    def address(self: Wallet, value: str) -> None:
+        """
+        Setter for address that reflects in classic_address.
+
+        Args:
+            value: New value for address/classic_address.
+        """
+        self.classic_address = value
+
     def __init__(
         self: Wallet,
-        seed: str,
-        sequence: int,
+        seed: Optional[str] = None,
+        sequence: Optional[int] = 0,
         *,
         algorithm: Optional[CryptoAlgorithm] = None,
+        master_address: Optional[str] = None,
     ) -> None:
         """
         Generate a new Wallet.
 
         Args:
             seed: The seed from which the public and private keys are derived.
-            sequence: The next sequence number for the account.
+                Generates a new seed if not included.
+            sequence: The next sequence number for the account. Defaulted to 0 if not
+                included.
             algorithm: The algorithm used to encode the keys. Inferred from the seed if
-                not included.
+                not included. The default is None.
+            master_address: Include if a Wallet uses a Regular Key Pair. It must be
```

Comment:
super nit: this should probably go before `algorithm`

Reply:
Put master address before algorithm? What's the reasoning?

Reply:
It's more likely to be used. Everything after the `*` must use a keyword, so order doesn't matter.

Like I said, super nit - if you don't want to, that's fine.